docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -410,6 +410,54 @@
                     </td>
                 </tr>
                 <tr>
+                                              <!-- change link to StandAlone design/tutorial doc -->
+                <th colspan="3" class="section" id="StandAlone"><a href="../index.html">StandAlone job configuration</a></th>
+                </tr>
+                <tr>
+                    <td class="property" id="coordinator.zk.connect">coordinator.zk.connect</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        <strong>Required </strong> for Zookeeper-based StandAlone applicaiton. Zookeeper coordinates to be used by
```

Comment:
typo in "application"

as before, let's not use "Standalone". It can just be "Required for Zookeeper-based coordination"